src/components/Form.js

```diff
@@ -48,6 +48,9 @@ const propTypes = {
     /** Should the button be enabled when offline */
     enabledWhenOffline: PropTypes.bool,
 
+    /** Whether the action is dangerous */
```

Comment:
I don't understand this - what does it mean for an action to be "dangerous"? 😕 

Why was this added to such a low level component..

Reply:
To me, "Dangerous" means if the user clicks the "Submit" button, the action being taken should only be taken with caution. I think it would be helpful if you can recommend a "better solution" or what you'd prefer seeing - I am guessing you are saying you'd like to see a more generic prop name, maybe something having to do with the submit button color?

Reply:
>To me, "Dangerous" means if the user clicks the "Submit" button, the action being taken should only be taken with caution.

>I am guessing you are saying you'd like to see a more generic prop name

No, I think it's _too_ generic. Which "action" is "dangerous"? It's a Form so the user can take many actions. It looks like this just controls the color of the button. So my suggestion would be something like `isSubmitButtonDangerous`, `isSubmitActionDangerous` etc.

Reply:
Aah ok I'm glad we got that clarified, I see what you mean that technically there can be many "actions" taken in a form, though I thought it was a bit self explanatory that the main "action" is submitting the form - but it's good to be clear 👍 I like your suggestions, I think `isSubmitActionDangerous` is a bit more clear to me - there should only be 1 "submit action" (clicking on the submit button) so I'll go with that - follow-up PR incoming!

Reply:
@marcaaron follow-up PR here: https://github.com/Expensify/App/pull/13981